Gemfile

```diff
@@ -48,6 +48,7 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'letter_opener', :group => :development
 end
```

Comment:
if you are adding this inside group :development, :test do scope, no need of using  :group => :development